src/argilla/client/feedback/schemas/metadata.py

```diff
@@ -43,7 +43,7 @@ class MetadataPropertySchema(BaseModel, ABC):
 
     Args:
         name: The name of the metadata property.
-        description: A description of the metadata property. Defaults to `None`.
+        title: A title of the metadata property (what's shown in the UI). Defaults to `None`.
```

Comment:
`description` was optional but `title` is not (in the same fashion with `fields` and `questions`). So probably you need to change the documentation, models and tests.

Reply:
Hmm ok, then we should default it internally to `title = name` if `title is None`?

Reply:
If that's what we are doing for fields and questions then I guess we can do the same. Can you confirm @frascuchon ?

Reply:
Yes, the same behavior that we have already for `fields`. Anyway, this should be reflected in the docstring